### ISSUE TITLE
BAD SIGNATURE when installing on Alpine Linux

### ISSUE THREAD

**Issue by gnd** [elastic/apm-agent-php#1149]
Hello, 

recently our Docker builds started to fail when trying to install APM agent from the APK package. The reason they fail is: 

```ERROR: apm-agent-php-1.12.0: BAD signature```

This is the case with Alpine 3.17, Alpine 3.18 and Alpine 3.19 and with APM agent versions 1.12 and 1.13. 

You can reproduce this by trying to build this Dockerfile: 
```
FROM docker.io/alpine:3.19 as base

RUN apk update && apk upgrade 
RUN apk add ca-certificates wget && wget -q https://github.com/elastic/apm-agent-php/releases/download/v1.13.0/apm-agent-php_1.13.0_x86_64.apk
RUN apk add --allow-untrusted apm-agent-php_1.13.0_x86_64.apk
```

Thanks for checking this out !

**Comment by apetrov1-godaddy** [comment]
I have the same situation

**Comment by zyv4yk** [comment]
maybe similar to this issue: https://gitlab.alpinelinux.org/alpine/apk-tools/-/issues/11000#note_394799

**Comment by zyv4yk** [comment]
yep, recheck with agent 1.10, this issue was solved in apk but still needs to be fixed before the release of apk-tools v3 as said in the apk-tool warning 
```
Support for packages without datahash will be dropped in apk-tools 3.
```
